### DISCUSSION TITLE
KeyChord cancel on wrong key

### DISCUSSION THREAD

**Discussion by gruvw** [qtile/qtile#4184] [Q&A] [answered]
Hello, is there a way to cancel the current `KeyChord` on "wrong" key press ?

I have the following:
```python
KeyChord([mod], "s", [
    Key([], "space", lazy.spawn(Commands.music_pause), desc="Pause music"),
    Key([], "l", lazy.spawn(Commands.music_next), desc="Next song"),
    Key([], "h", lazy.spawn(Commands.music_previous), desc="Previous song"),
], name="Music"),
```

And I would like any other key (other than `space`, `l` and `h`) to "act like" escape, essentially cancelling the key chord mode.

Thx :)

**Comment by m-col** [comment, marked as answer]
I'm afraid this isn't possible. You would need to add `Key` entries in your list above for all other keys and bind them to `lazy.ungrab_chord()`. This might sound a bit ugly but to be honest that's how it would need to be implemented internally if it was supported (at least in X11), so maybe you could hide the ugliness in little helper function to automatically generate all the other `Key` values.

**Reply by gruvw** [reply]
Ok thanks, however how to loop over every other possible keys ?
